spec/std/enum_spec.cr

```diff
@@ -134,10 +134,10 @@ describe Enum do
     end
 
     it "for flags enum" do
```

Comment:
Perhaps also add a spec for value `0` again (== `eq None`).

Reply:
yess good idea